tests/tests.lisp

```diff
@@ -256,22 +263,22 @@
 
 ;;; transformations
 
-(deftest fill! (tests)
+(deftest fill! (transformations)
   (let ((a (make-array '(3 2) :initial-element 1.0)))
     (aops:fill! a 0)
     (assert-equalp a #2A((0.0 0.0)
                          (0.0 0.0)
                          (0.0 0.0)))))
 
-(deftest coercing (tests)
+(deftest coercing (transformations)
   (assert-equality (curry #'every #'eql)
       #(1d0 2d0 3d0)
       (map 'vector (aops:coercing 'double-float) #(1 2 3)))
   (assert-equality (curry #'every #'eql)
       #(1d0 4d0 9d0)
       (map 'vector (aops:coercing 'double-float (lambda (x) (* x x))) #(1 2 3))))
 
-(deftest generate (tests)
+(deftest generate (transformations)
```

Comment:
This one probably comes under `creation` since it doesn't take an array as input.

Reply:
Ah I see it's by file. That makes sense, but perhaps the `generate` function is in the wrong place.

Reply:
Yeah, I agree.

I think we also move the function definition to "creation.lisp"; it's now in "transformations.lisp"

I'll make that change in the PR and re-up.

Reply:
Done. See comment below.